main.js

```diff
@@ -15,8 +15,18 @@ var accomplishInput = document.querySelector('#accomplishInput');
 var minutesInput = document.querySelector('#minutesInput');
 var secondsInput = document.querySelector('#secondsInput');
 var startActivityButton = document.querySelector('.start-button');
+var warningCategory = document.querySelector('.warning-category');
+var warningDescription = document.querySelector('.warning-description');
+var warningMinutes = document.querySelector('.warning-minutes');
+var warningSeconds = document.querySelector('.warning-seconds');
+var countdownMinutes = document.querySelector('.countdown-minutes');
+var countdownSeconds = document.querySelector('.countdown-seconds');
 
-
+var categoryIsClicked = {
+  studySelected: false,
+  meditateSelected: false,
+  exerciseSelected: false
+};
```

Comment:
Great idea to use an object!! As discussed on the call, since this is an unwanted global variable, we'll need to refactor, but let's keep this object in mind. Maybe we can make a Category class at some point?